Update mention of coverage config file to `pyproject.toml` in documentation (#4816)
cookiecutter/cookiecutter-django: docs/testing.rst

   At the root of the project folder, you will find the `pytest.ini` file. You can use this to customize_ the ``pytest`` to your liking.

>    There is also the `pyproject.toml`. This is the configuration file for the ``coverage`` tool. You can find out more about `configuring`_ ``coverage``.

Reviewer: ```suggestion
   The configuration for ``coverage`` can be found in ``pyproject.toml``. You can find out more about `configuring`_ ``coverage``.
```
